drvregistry/drvregistry.go

```diff
@@ -37,7 +37,7 @@ type DrvRegistry struct {
 // Functors definition
 
 // InitFunc defines the driver initialization function signature.
-type InitFunc func(driverapi.DriverCallback, map[string]interface{}) error
+type InitFunc func(driverapi.DriverCallback, interface{}, map[string]interface{}) error
```

Comment:
Would it make sense to have the plugin getter be a member of `DrvRegistry` ?
At the end all the remote drivers register with the registry, he can hold the plugin getter for them to be called in the init function.